src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import CalculatorApp from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Kindly change your component to be class-based, this is a project requirement.